src/Gui/TaskView/TaskView.h

```diff
@@ -199,6 +205,8 @@ protected Q_SLOTS:
     QSint::ActionPanel* taskPanel;
     TaskDialog *ActiveDialog;
     TaskEditControl *ActiveCtrl;
+    bool restoreWidth = false;

```

Comment:
It seems that this new feature is deactivated by default.

The boolean Preferences/DockWindows/TaskView/RestoreWidth has to be manually set to true.